CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to AccessKit
+
```

Comment:
A welcoming message, perhaps including something about what types of contributions you are looking for / open to would be fitting to have here. 

Felt like it would be disingenuous if I tried to write one as I'm not a regular maintainer.